### ISSUE TITLE
[Enhancement] A11y improvements for Segmented Control

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#3687]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
To make Segmented Control support proper announcing to screen readers we need to:
- [x] ~~Announce total number of Segment tabs (1/1 -> 1/3)~~
   [This is a known bug in Chrome with VoiceOver on macOS](https://bugs.chromium.org/p/chromium/issues/detail?id=1405462)
- [x] Announce selected tab
- [x] Implement [Tabs pattern from ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)

## Are there any additional context?
Also filed a bug issue in the Ionic repo: ionic-team/ionic-framework#30022 

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [x] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [x] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [x] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
